### DISCUSSION TITLE
Is it possible to request a run at a particular environment version?

### DISCUSSION THREAD

**Discussion by AlabasterAxe** [actions/virtual-environments#5899] [General questions] [answered]
It looks like the most recent image deployment of ubuntu-18.04 (20220710.1) is breaking some of our tests and I'd like to confirm this by explicitly specifying the version of ubuntu-18.04 to use. Is such a thing possible? I dug through some of the Actions docs but I didn't see any way to request a specific version of the environment. Thanks!

**Comment by panekj** [comment, marked as answer]
It's not possible
